shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
I don't see a reason we have to do such a change to documentation as I know it will affect a few programs. Consider this a v4 change, so for now lets just keep the 3 card names on it